src/modm/driver/touch/touch2046_impl.hpp

```diff
@@ -0,0 +1,79 @@
+// coding: utf-8
+/*
+ * Copyright (c) 2021, Raphael Lehmann
+ *
+ * This file is part of the modm project.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+// ----------------------------------------------------------------------------
+
+#ifndef MODM_TOUCH2046_HPP
+	#error	"Don't include this file directly, use 'ltc2984.hpp' instead!"
+#endif
```

Comment:
Maybe we should just get rid of this and use `#pragma once` everywhere. This is just an additional source of error, nobody will include the `_impl.hpp` file anyways.